scripts/generate-config-doc.js

```diff
@@ -20,8 +20,18 @@ function getGitUsername() {
 
 const configContent = readFileSync(join(__dirname, "..", "defaults", "config.js"), "utf8");
 
+const docRoot = process.argv[2];
+
+if (!docRoot) {
+	log.error("Missing DOC_ROOT_PATH. Pass the path to the cloned `thelounge.github.io` repo.");
+	process.exit(1);
+}
+
 const docPath = join(process.argv[2], "_includes", "config.js.md");
 
+/** @type {string[]} */
+const acc = [];
```

Comment:
I don't use JSDoc type annotations for TypeScript often. If there is a way to do this more concisely without separately defining and annotating `acc` like this, it would be better imo. I didn't/couldn't just convert this file to `.ts` due to reasons explained in the PR comment.